3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -422,6 +422,54 @@ I membri dell'organizzazione sono prestabiliti. Qualora le competenze interne al
 
 Non sono previste variazioni della composizione dell'organizzazione, se non in via straordinaria e comunque discussa preventivamente con il Committente.
 
+=== Processo di gestione della qualità <processo_gestione_qualità>
+
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.2.5_
+
+==== Scopo
+
+Il processo di gestione della qualità ha lo scopo di garantire che i prodotti del progetto soddisfino gli obiettivi di qualità del gruppo e i bisogni del Proponente.
+
+=== Attività
+==== Pianificazione
+
+Il gruppo #err418 pone i seguenti principi di qualità:
+
++ tutto il materiale presente nei branch principali della repository (`src` e di conseguenza `main`) deve essere di buona qualità:
+  - tutti i prodotti del gruppo devono essere gestiti tramite repository su GitHub (@repository-github), nelle quali deve essere implementato il sistema di _featuring branching_;
+  - ogni modifica che vuole essere introdotta in un prodotto dovrà essere prima approvata dal Verificatore tramite il meccanismo di pull request (@controllo_release).
++ il contenuto della documentazione deve essere scritto in modo chiaro e comprensibile;
++ la documentazione prodotta deve seguire degli standard interni e/o esterni:
+  - standard interni:
+    - vengono definiti nel presente documento alla @norme-documentazione.
+  - standard esterni:
+    - il gruppo adotta i seguenti standard ISO:
+    #figure(table(
+      align: left,
+      columns: 2,
+      [*Standard ISO*], [*Documento*],
+      [12207:2017], [#ndp, #adr],
+      [9126 1:2001], [#pdq],
+    ), caption: [Tracciamento ISO-documenti])
+    In particolare, per ogni processo interno al presente documento viene specificato il grado di conformità allo standard ISO/IEC/IEEE 12207:2017 all'interno di una sezione dedicata (@tracciamento_paragrafi).
++ il codice prodotto deve seguire degli standard interni, definiti nel presente documento (#TODO);
++ la quantità di errori individuati nei prodotti del progetto, sia documentali che software, al momento in cui questi sono presenti nei branch principali della repository deve essere pari a 0;
++ il prodotto software deve soddisfare i bisogni e le richieste del Proponente;
```

Comment:
estenderi in "il prodotto software deve soddisfare i bisogni e le richieste del Proponente, identificate nel #adr e catalogate in: obbligatorie, desiderabili e opzionali."